Verilog/README.md

```diff
@@ -0,0 +1,26 @@
+# Hello World in Verilog
+## Installing Iverilog
+
+### Ubuntu/Linux
+
+* Update the local repository cache
+
+        $ sudo apt-get update
+
+* Install (update) the Icarus Verilog package
+
+        $ sudo apt-get install verilog
```

Comment:
Enclose the code in back ticks(`). Check out this [link](https://help.github.com/articles/creating-and-highlighting-code-blocks/) for more information!